### PR TITLE
Improving Docs for Data Science ML and python quick starters

### PR DESCRIPTION
**Pull request by hugowschneider**

### CODE REVIEW

File: boilerplates/be-python-flask/README.md [hunk 1]
```diff
@@ -6,7 +6,26 @@ Use this quickstarter when you want to build a [python](http://flask.pocoo.org/d
...
 ## What files / architecture is generated?
-
+```
+.
+├── Jenkinsfile  -  This file contains Jenkins build configuration settings
+├── README.md  -  Readme file template
+├── docker-compose.yml  -  For running all services locally
+├── jenkinsfile_helper.py  -  Helper function used during the Jenkins pipeline
+├── build.sh  -  Build Script. This script copies all necessary files to the docker images 
+├── docker-prediction  -  This folder contains Docker configuration settings for prediction service
```
Comment (gerardcl) [inline boilerplates/be-python-flask/README.md]: this is not the right file/folder structure, this is the one related to ds-ml-service
@hugowschneider let me know if you want me to update it.

File: boilerplates/ds-ml-service/files/README.md [hunk 2]
```diff
@@ -1,231 +1 @@
...
-
-| Environment Variable | Description | Allowed Values |
-|------|-------|---|
-| DSI_DEBUG_MODE | Enables debug mode | true, 1 our yes for debug mode, otherwise debug is disasbled|
-| DSI_TRAINING_BASE_URL | The base url where the prediction service should get the model from | url (e.g. https://training.OpenShift.svc |
-| DSI_TRAINING_SERVICE_USERNAME | Username of the training service | string, required |
-| DSI_TRAINING_SERVICE_PASSWORD | Password of the training service | string, required |
-| DSI_PREDICTION_SERVICE_USERNAME | Username to be set as default username for accessing the service | string, required |
-| DSI_PREDICTION_SERVICE_PASSWORD | Password to be set as default password for accessing the service | string, required |
+# Data Science Industrialization Service for ...
```
Comment (gerardcl) [inline boilerplates/ds-ml-service/files/README.md]: what if we mv/cp the readme to files through build.sh script when provisioning the component so the users can have the whole readme too in their repos at the beginning?

**Comment by clemensutschig** [comment]
@hugowschneider  - what do we do with this beast here?

**Comment by gerardcl** [comment]
closed by https://github.com/opendevstack/ods-project-quickstarters/pull/229

**Comment by gerardcl** [comment]
closed by #229

**Pull request closed without merging**